modules/ROOT/pages/authentication-authorization/privileges-reads.adoc

```diff
@@ -67,6 +74,14 @@ For example, we can disable users with the role `regularUsers` from finding all
 DENY TRAVERSE ON HOME GRAPH NODES Payments TO regularUsers
 ----
 
+Although we just granted the role `regularUsers` the right to read all properties on nodes with label `Post`,
+we may want to make this more fine-grained using xref:authentication-authorization/property-based-access-control.adoc[Property-based access control] to hide the posts with `secret` property set to `true`.
+The following example shows how to do that:
+
+[source, cypher, role=noplay]
+----
+DENY TRAVERSE ON HOME GRAPH FOR (:Post {secret: true}) TO regularUsers
```

Comment:
no tiny examples for read and match?

Reply:
I had in a previous commit. Would you like me to add them back? I removed them when we created the separate Property-based access control page since I thought I was repeating myself a lot.

Reply:
I think it might be nice to have one each here (and then maybe more than one for each in the main property rule page)